crates/milli/src/update/new/extract/searchable/tokenize_document.rs

```diff
@@ -265,6 +276,10 @@ mod test {
                 2,
                 16,
             ]: "catto",
+            [
+                3,
+                0,
+            ]: "unsearchable",
```

Comment:
why did this test change?

Reply:
The check to know if a field is searchable is made a bit latter in the code, so this test no more have the searchable field filter